eZ/Bundle/EzPublishSolrSearchEngineBundle/EzPublishSolrSearchEngineBundle.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
- * File containing the EzPublishSolrBundle class.
+ * This file is part of the eZ Publish Kernel package
  *
  * @copyright Copyright (C) eZ Systems AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  * @version //autogentag//
  */
 
-namespace eZ\Bundle\EzPublishSolrBundle;
+namespace eZ\Bundle\EzPublishSolrSearchEngineBundle;
```

Comment:
Documented in https://github.com/ezsystems/ezpublish-kernel/commit/52339375ad55549230dd3443a3512765aa99fc1f.